Manifests/ManagedPreferencesApplications/com.1password.1password.plist

```diff
@@ -7,15 +7,15 @@
 	<key>pfm_app_url</key>
 	<string>https://1password.com</string>
 	<key>pfm_description</key>
```

Comment:
Let's also add `pfm_app_min` to the root-level manifest so we're documenting formally that this manifest requires v8.

Given when MDM support was added we probably should add `pfm_add_min` to the individual preferences as well so if and when new management features are added we can build off of that.